docs/annotate.md

```diff
@@ -31,6 +31,19 @@ Including at least the following annotations is recommended:
   * Description (<a href="http://purl.org/dc/terms/description" target="_blank">`dcterms:description`</a>)
   * License (<a href="http://purl.org/dc/terms/license" target="_blank">`dcterms:license`</a>)
 
+If the `--interpolate` option is used, then some placeholders, of the form `%{NAME}`, can be used within the annotation values and will be automatically replaced by computed values. Currently supported placeholers are:
+
+  * `%{ontology_iri}`, replaced by the ontology's IRI;
+  * `%{version_iri}`, replaced by the ontology's version IRI.
+
+Example:
+
+    robot annotate --input fbcv-module.owl \
+      --interpolate \
```

Comment:
I dont think there is a precedence for using option flags in ROBOT, I am pretty sure the general strategy is to always use values (`--interpolate true`) however redundant.

Reply:
I’ve been wondering about that.

ROBOT is not always consistent on the issue, even within the context of a single command. For example, the `annotate` command has both the `--remove-annotation` flag, which does not accept a value, and the `--annotate-derived-from <true|false>` and `--annotate-defined-by <true|false>`.

Happy to follow the “general strategy”… assuming that there _is_ one, and that it is stated somewhere.

Reply:
> ROBOT is not always consistent on the issue

I looked at that again, and I think I had exaggerated the inconsistency. It is clear that the vast majority of boolean-type options in ROBOT take a `true|false` argument – in fact only `annotate` and `template` also accept a few argument-less “flags”.

I’ll update the PR to make `--interpolate` take a `true|false` argument.

(Of course this point becomes moot if we decide instead that there is no need for an option and that interpolation should be unconditionally enabled.)

Reply:
While there are some exceptions that we maintain for backwards compatibility (aka "mistakes"), I do want all new CLI options to take an argument. This has been a hard-won lesson: by including a boolean argument in the beginning, we leave room for additional options down the road. Thanks!